plostd.c

```diff
@@ -34,34 +34,27 @@ int plostd_isalnum(char c)
 }
 
 
-int plostd_strlen(char *s)
+int plostd_strlen(const char *s)
```

Comment:
better will be
```
unsigned int plostd_strlen(const char *s)
```
returned value `k` is type `unsigned int`

Reply:
sure